service/tests/service.test.js

```diff
@@ -0,0 +1,62 @@
+import { expect, test } from 'vitest'
+
+import dotenv from 'dotenv'
+
+dotenv.config();
+
+const authServerURL = import.meta.env.VITE_BASE_URL + ':' + import.meta.env.VITE_AUTH_SERVER_PORT;
+const serviceURL = import.meta.env.VITE_BASE_URL + ':' + import.meta.env.VITE_SERVICE_PORT;
+
+
+test('givenValidToken_whenGet_thenObtainedData', async () => {
+    const credentials = {
+      login: 'user@email.com',
+      password: '12345678',
+    };
+  
+    const authResponse = await fetch(`${authServerURL}/login`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(credentials),
```

Comment:
A a side note, no fetch should be issued without a timeout (every fetch in the code. I mean, not just this one).

```
fetch(url, {
   // options...
   signal: AbortSignal.timeout(2000)
})
```

It's not too problematic in test suites because vite timeouts the test execution (after 4 seconds maybe... 🤔)